swift/Package.swift

```diff
@@ -1,19 +1,3 @@
-/**
- *    Copyright (c) 2019 Uber Technologies, Inc.
```

Comment:
this confuses parsing toolchain version when running swift build

Reply:
Since this is a generic file, not having the license should be okay.